### ISSUE TITLE
NVDA causing CHTMLView Applications not to exit

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3306]
**Reported by ScottV on 2013-06-27 18:41**
Applications developed with the Microsoft Foundation Class (MFC) CHTMLView will not exit when NVDA is running. When NVDA is exited the MFC CHTMLView application exits with out issues.

Sample CHTMLView application:
http://msdn.microsoft.com/en-us/library/ms177540(v=vs.80).aspx

The Application Window closes but the applications process remains running. NVDA seems to be hanging the main thread of the application Window with an Access Vialoation.

When NVDA is running there is an Unhandled Exception (0xC0000005) Access Violation Reading Memory Location when the application is closed.

When NVDA is not running there is no error when exiting.

What mechanism is NVDA using to read the text from the applications in order for the application to signal NVDA that it is closing, so the applcation or NVDA can release the memory to avoid the error?

**Comment by bhavyashah** [comment]
According to @michaelDCurran's https://github.com/nvaccess/nvda/issues/3306#issuecomment-155307840, this issue seems to take place with Narrator as well, suggesting that it isn't an inherent NVDA problem. Additionally, Mick requested the original author of this ticket, who is now out-of-reach, to perform testing with Narrator and revert with results, which unfortunately hasn't been done. Unless Mick has any feedback or recommends otherwise, I suggest closing. @ehollig

**Comment by Adriani90** [comment]
@michaelDCurran is there any update on this issue?

**Comment by Adriani90** [comment]
@ScottV is this still reproducible with NVDA 2019.1.1?

**Comment by leonardder** [comment]
I'm closing this because of lack of response. If this is still an issue, we can always reopen.
